code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -101,8 +101,8 @@
 	tastes = list("crust" = 1, "tomato" = 1, "cheese" = 2, "pasta" = 1)
 
 /obj/item/reagent_containers/food/snacks/macpizzaslice
-	name = "mac n cheese pizza slice"
-	desc = "A delicious slice of pizza topped with macaroni cheese... wait, what the hell? Who would do this?!"
+	name = "macaroni & cheese pizza slice"
+	desc = "A delicious slice of pizza topped with macaroni & cheese... wait, what the hell? Who would do this?!"
```

Comment:
Mac n Cheese is a very popular dish in the US - with a very long history. And I am quite sure that Mac n Cheese pizza is predominantly an American thing. So the American spelling should be retained. 

(Please revert this line of change)

Reply:
That's fair, I'm not american, I'm canadian, and we just call it kraft dinner or kd half the time, so I'll swap it back!